src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/CreateFinancialTransactionDTO.java

```diff
@@ -0,0 +1,12 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionType;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+public record CreateFinancialTransactionDTO(@Min(1) @NotNull Long walletId, @DecimalMin("0.0") BigDecimal amount,
```

Comment:
Please refactor annotation `@DecimalMin("0.0")` to `@DecimalMin("0")`
Please add annotation `@Digits("13,2")`